.gitmodules

```diff
@@ -17,3 +17,6 @@
 [submodule "lib/base-contracts"]
 	path = lib/base-contracts
 	url = https://github.com/base-org/contracts
+[submodule "lib/solady"]
+	path = lib/solady
+	url = https://github.com/Vectorized/solady
```

Comment:
I just checked, we actually don't need this.

I might want to add it in a subsequent PR, but will remove for now.

Reply:
did that

Reply:
OK, now it seems like we do need it, the updated base dep depends on it.